File: src/main/java/com/erisu/cloud/megumi/command/CommandType.java
package com.erisu.cloud.megumi.command;

import lombok.AllArgsConstructor;
import lombok.Getter;


/**
 * 就两个不用工厂-策略模式吧，再说
 */
@Getter
@AllArgsConstructor
public enum CommandType {
    FRIEND(0,"友達"),GROUP(1,"群聊");
    private Integer type;
    private String typeName;

}

File: megumi/src/main/java/com/erisu/cloud/megumi/command/CommandType.kt
package com.erisu.cloud.megumi.command

/**
 *@Description
 *@Author alice
 *@Date 2021/6/23 18:16
 **/
enum class CommandType(val type: Int, val typeName: String) {
    FRIEND(0, "友達"), GROUP(1, "群聊");
}